Competition2018/src/main/java/competition/subsystems/climberdeploy/ClimberdeploySubsystem.java

```diff
@@ -41,14 +58,18 @@ public void stopClimberArm() {
 	 * speeds up the arm, regardless of what direction the arm is moving
 	 */
 	public void increaseSpeed() {
```

Comment:
Let's simplify this whole increase/decrease speed thing into something a little simpler: a slow and a fast mode.

have a new variable called "currentDeploySpeed", and then two doubleproperties: "slowDeploySpeed" and "fastDeploySpeed". Then we can use this increase/decrease speeds to change the current speed to either fast or slow.

Reply:
So this means deploy/detract speed would be determined by some sort of some sort of controller that might look like the attached image, rendering the Increase Speed and Decrease Speed Commands obsolete. This is my current understanding. I've added the two suggested double properties but have not deleted Increase Speed and Decrease Speed Commands.
![5436578965](https://user-images.githubusercontent.com/16183605/35179964-6171f17c-fd5a-11e7-91dd-361e718f58dd.png)